ChangeLog

```diff
@@ -34,6 +34,11 @@ Release date: Undefined
 ..
   Put bug fixes that will be cherry-picked to latest major version here
 
+* Enable functional tests for optional checkers by evaluating ``load-plugins`` in the test option file.
+  (This only concerns tests of the Pylint codebase itself).
+
+  Closes #4331
+  
```

Comment:
You don't need to add a ChangeLog entry for pylint internals that don't have consequences for the end user.

Reply:
Sure! 
Added the tests, and removed the changelog entry.